cmd/commands/run/command.go

```diff
@@ -45,7 +45,9 @@ func NewCommand() *cli.Command {
 			return di.Node.Wait()
 		},
 		After: func(ctx *cli.Context) error {
-			return di.Node.Kill()
+			err := di.Node.Kill()
+			di.Shutdown()
```

Comment:
Shutdown could kill the 'Node'.

Reply:
Currently `di.Bootstrap()` doest not do start of node, so would be strange to Stop what it did not started.
We can do it when we dont use `Bootstrap` for service entrypoint (`myst service`), which has different start/stop flow